curvefs/src/metaserver/metastore.cpp

```diff
@@ -490,10 +490,14 @@ MetaStatusCode MetaStoreImpl::CreateInode(const CreateInodeRequest* request,
 
 MetaStatusCode MetaStoreImpl::CreateRootInode(
     const CreateRootInodeRequest* request, CreateRootInodeResponse* response) {
-    uint32_t fsId = request->fsid();
-    uint32_t uid = request->uid();
-    uint32_t gid = request->gid();
-    uint32_t mode = request->mode();
+    InodeParam param;
+    param.fsId = request->fsid();
+    param.uid = request->uid();
+    param.gid = request->gid();
+    param.mode = request->mode();
+    param.length = 0;
+    param.type = FsFileType::TYPE_DIRECTORY;
+    param.rdev = 0;
```

Comment:
param.parent not inited, it may any num in release mode

Reply:
> param.parent not inited, it may any num in release mode

fixed.